CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.25) # Debian 11 Ubuntu 24.04 Fedora 36
 
-option(WITH_FFMPEG_PLAYER "Enable support for FFMPEG player" ON)
+option(WITH_FFMPEG_PLAYER "Enable support for FFMPEG player. Note: This is useless for Qt >= 6.8" ON)
```

Comment:
DEFAULT to `OFF`?

Reply:
I believe there shouldn't be a default value.

For Linux users/packagers and people who build from source, they should always specify all options to avoid upstream flag changes.

On our side (Win/macOS build), it doesn't matter.

If we change it, it will silently change someone's build.

We should have set them all to OFF by default, but it is already what it is, so let's just don't change it.

Reply:
ok for now.

maybe remove the ffmpeg related code in a later time.  as it has already been the actual backend of qmultimedia.